Addons/DataToColor/Query.lua

```diff
@@ -80,23 +80,15 @@ function DataToColor:GetTargetName(partition)
     return 0
 end
 
-function DataToColor:CastingInfoSpellId()
-    local _, _, texture, _, _, _, _, _, spellID = CastingInfo()
```

Comment:
This function differs in Classic and TBCC. https://wowpedia.fandom.com/wiki/API_UnitCastingInfo

So in Classic version this simply not gonna work(spellID will be always nil) and most likely cripples the `CastingHandler` which keeps track of `PlayerReader.IsCasting` flag.

I've looked around, there's a quite popular lib which solves the issue like this by checking if the `name` is exists.
`https://github.com/oUF-wow/oUF/blob/5ceda298405595ef2ba6d2903281fbe60ac95867/elements/castbar.lua#L109`